__test__/shop.test.js

```diff
@@ -0,0 +1,89 @@
+var Shop = require('../src/shop');
+var Item = require('../src/item');
+
+describe("Gilded Rose", function() {
+  it('Name does not change after update', () => {
+    const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
+    const items = gildedRose.updateQuality();
+    expect(items[0].name).toBe("foo");
+  });
+
+  it('Quality degrades twice as fast after sellIn is zero', () => {
```

Comment:
Can we also have a test for quality where there is still a positive sellIn value? We should have a test to cover all cases.

Reply:
Ok. I'll add a test.